### ISSUE TITLE
48in24 Approaches for Reverse String

### ISSUE THREAD

**Issue by michalporeba** [exercism/elixir#1420]
I'd like to propose a few approaches to the reverse string exercise as requested [on the forum](https://forum.exercism.org/t/48in24-exercise-01-30-raindrops/9247/4) for the 48in24 challenge.

**Comment by angelikatyborska** [comment]
Did you mean to write _raindrops_ instead of _reverse string_?

**Comment by michalporeba** [comment]
Of course. I wrote it after I had just finished working on a reverse string
for the Clojure track. I mean the raindrops.
